docs/testing_data_migrations.md

```diff
@@ -5,6 +5,6 @@ Data migrations can/should be tested using production data. To download a copy f
 1. `cf login -a https://api.london.cloud.service.gov.uk --sso`
 2. Set space to `bat-prod`
 3. Make sure conduit is installed `cf install-plugin conduit`
-4. Download the db `cf conduit register-postgres-production -- pg_dump -f register-postgres-production.sql --no-acl --no-owner --clean`
+4. Download the db `cf conduit register-postgres-13-production -- pg_dump -f register-postgres-production.sql --no-acl --no-owner --clean`
```

Comment:
13 is the old db. Pre-migration to postgres 14.

Reply:
hmm, I wasn't able to download the other one, only this 13 one. I'll change it back.

Reply:
done